backend_new/src/interceptors/activity-log.interceptor.ts

```diff
@@ -73,8 +74,11 @@ export class ActivityLogInterceptor implements NestInterceptor {
   intercept(context: ExecutionContext, next: CallHandler) {
     const { module, action, user, activityLogMetadata } =
       this.getBasicRequestInfo(context);
-
-    if (action === permissionActions.read) {
+    if (
+      action === permissionActions.read ||
+      action === permissionActions.submit
```

Comment:
it is a little different from the old backend, since before we had separate controllers for applications and application-submissions we just used a different guard, but with all the combining this felt like a better solution

since only application submission from public and application verification from public use the submit action I think this is an okay route to take but lemme know if you feel differently!